src/main/java/com/fabriccommunity/spookytime/registry/SpookyFeatures.java

```diff
@@ -0,0 +1,23 @@
+package com.fabriccommunity.spookytime.registry;
+
+import com.fabriccommunity.spookytime.SpookyTime;
+import com.fabriccommunity.spookytime.world.feature.LargeSkeletalTreeFeature;
+import com.fabriccommunity.spookytime.world.feature.SmallSkeletalTreeFeature;
+import net.minecraft.util.registry.Registry;
+import net.minecraft.world.gen.feature.DefaultFeatureConfig;
+import net.minecraft.world.gen.feature.Feature;
+import net.minecraft.world.gen.feature.FeatureConfig;
+
+/**
+ * @author Indigo Amann
+ */
+public class SpookyFeatures {
+    public static Feature<DefaultFeatureConfig> SMALL_SKELETON_TREE = register("skeleton_tree_small", new SmallSkeletalTreeFeature(DefaultFeatureConfig::deserialize, false));
+    public static Feature<DefaultFeatureConfig> LARGE_SKELETON_TREE = register("skeleton_tree_large", new LargeSkeletalTreeFeature(DefaultFeatureConfig::deserialize, false));
+    public static void init() {
+
+    }
+    public static <C extends FeatureConfig, F extends Feature<C>> F register(String name, F feature) {
```

Comment:
Newline here.

Reply:
Are you saying it needs one here, or there is one here when there shouldn't be?

Reply:
I'm saying there should be one. Methods should always have whitespace between each other.